index.html

```diff
@@ -164,11 +164,11 @@ <h3>Profesional Art Printing Data</h3>
       <div class="about-content">
         <article class="about-me">
           <h2>About me</h2>
-          <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of
+          <artcile>Hello I'm a software developer! I can help you build a product , feature or website Look through some of
```

Comment:
- [ ] Since this is a paragraph, kindly use a `p` tag instead.
- [ ] Please follow the Figma design guidelines for your chosen template. Currently, there's a mismatch in color and font.

## Figma Design
![image](https://user-images.githubusercontent.com/92215878/220968855-134876f1-5045-4bda-8366-6a89ac36ba1f.png)

## Your Design
![image](https://user-images.githubusercontent.com/92215878/220968591-8f30731f-8e00-476a-8911-ad93f8652a7e.png)